src/tokenizer/tokenizer.rs

```diff
@@ -260,3 +261,21 @@ pub trait TokenFilter<TailTokenStream: TokenStream>: Clone {
     /// Wraps a token stream and returns the modified one.
     fn transform(&self, token_stream: TailTokenStream) -> Self::ResultTokenStream;
 }
+
+#[cfg(test)]
+mod test {
+    use super::Token;
+
+    #[test]
+    fn clone() {
```

Comment:
Maybe test other attributes or remove the test entirely?

Reply:
Added. Thank you.